packages/common/src/utils/index.ts

```diff
@@ -32,3 +32,4 @@ export * from "./isComboboxItemSelected";
 export * from "./wait";
 export * from "./IUserHubSettings";
 export * from "./IUserSiteSettings";
+export * from "./_array";
```

Comment:
we should not export from `_array` - the `_` indicates that these are internal only funcitons.

please add an `array.ts` file and move your new util there.